source/fundamentals/object-serialization.txt

```diff
@@ -0,0 +1,241 @@
+====================
+Object Serialization
+====================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+**Object serialization** is the process of translating data between two
+mediums. MongoDB stores data as :manual:`MongoDB Extended JSON
+<reference/mongodb-extended-json/>` containing fields that correspond
+with BSON data types. The driver uses object serializatio to translate
+data between Extended JSON and Java objects. You can specify how the
+driver should perform serialization using **codecs**.
+
+Codecs implement translation between Extended JSON and Java objects
+using methods that encode and decode data.
+Each ``CodecRegistry`` contains a list of codecs. When you specify a 
+``CodecRegistry`` for a ``MongoClient``, ``MongoDatabase``, or
+``MongoCollection``, the ``CodecRegistry`` is used for all subsequent
```

Comment:
Nitpick: repeated use of the same noun, consider: "the CodecRegistry" -> "it"

Reply:
Reworded instead to reduce this to just one usage in a sentence (thought "it" was unclear)